target/linux/ipq807x/files/arch/arm64/boot/dts/qcom/ipq8074-sxk80.dtsi

```diff
@@ -0,0 +1,635 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+/dts-v1/;
+
+#include "ipq8074.dtsi"
+#include "ipq8074-hk-cpu.dtsi"
+#include <dt-bindings/gpio/gpio.h>
```

Comment:
You can include the NSS DTSI here if you want it to work

Reply:
Dine